docs/extensions/httpfs.md

```diff
@@ -1,29 +1,28 @@
 ---
 layout: docu
-title: httpfs
+title: httpfs Extension
 ---
 
 The `httpfs` extension is an autoloadable extension implementing a file system that allows reading remote/writing remote files. For plain HTTP(S), only file reading is supported. For object storage using the S3 API, the `httpfs` extension supports reading/writing/globbing files.
 
 The `httpfs` extension will be, by default, autoloaded on first use of any functionality exposed by this extension.
 If you prefer to explicitly install and load this extension, you can always run `INSTALL httpfs` on first use and issue `LOAD httpfs` at the start of every session.
 
-## HTTP(S)
+## Running Queries over HTTP(S)
 
-With the `httpfs` extension, it is possible to directly query files over the HTTP(S) protocol. This currently works for CSV, JSON, and Parquet files.
+With the `httpfs` extension, it is possible to directly query files over the HTTP(S) protocol. This currently works for CSV, JSON, and Parquet files, and provides read-only access.
```

Comment:
```suggestion
With the `httpfs` extension, it is possible to directly query files over the HTTP(S) protocol. This works for all files supported by DuckDB or its various extensions, and provides read-only access.
```

Currently the only non-working format are sqlite databases via sqlite_scanner (but PR is up to fix it in sqlite_scanner repo)